### ISSUE TITLE
Simple cross-linking of module references.

### ISSUE THREAD

**Issue by mkrussel77** [Kotlin/dokka#1537]
**Is your feature request related to a problem? Please describe**
It would be helpful if the dokkaHtmlMultiModule task had a good way to produce relative links for elements between modules.
Currently one needs to manually add the `externalDocumentLink` to a module for each module it depends on.  One of the problems with this is that it cannot be a relative link.

**Describe the solution you'd like**
I would like the `dokkaHtml` task to auto have `externalDocumentLink` to same modules in the same parent project that they have an api dependency.  And that this would auto generate relative links to them.  Maybe have something like an `internalDocumentLink`.

If not an automated solution, then something that would allow relative links in the `externalDocumentLink`.

**Describe alternatives you've considered**
I tried using a relative link, but could not get that to work.

```kotlin
externalDocumentationLink {
    url.set( URL("../../$projectName/$projectName"))
}

externalDocumentationLink {
    url.set( URL("http://../../$projectName/$projectName"))
}
```

First one throws an exception and the second one produces wrong links.

http://../projectName/projectName/package/class/index.html

The only solution is to hard code the path to where it will be hosted.  This has the problem of needing to regenerate the documentation if root URL for the documentation changes.  And you have to build a toggle to switch to change between production hosting and developer testing.

**Comment by CommanderTvis** [comment]
That's a very vexing problem.

**Comment by kamildoleglo** [comment]
Fixed in `1.4.30` with the new multi-module documentation based on templates. No external links are necessary
